WRO-8722: Update in-editor linting guide (#296)
enactjs/cli: README.md

Ever since ESLint 6, global installs of ESLint configs are no longer supported.
To work around this new limitation, while still supporting in-editor linting, we've created a new [eslint-config-enact-proxy](https://github.com/enactjs/eslint-config-enact-proxy) package.
The [eslint-config-enact-proxy](https://github.com/enactjs/eslint-config-enact-proxy) acts like a small proxy config, redirecting ESLint to use a globally-installed Enact ESLint config.
In order for in-editor linting, `eslint-config-enact-proxy` should be installed locally on a project:

```sh
npm install --save-dev eslint-config-enact-proxy
```

> And you should modify `package.json` or `.eslintrc` in the project's root, change the following lines:

Reviewer: ```suggestion
Also, you need to modify `eslintConfig` property in `package.json`:
```

Author: OK
